bot/utils/messages.py

```diff
@@ -98,7 +98,14 @@ async def wait_for_deletion(
         try:
             await bot.instance.wait_for('reaction_add', check=check, timeout=timeout)
         except asyncio.TimeoutError:
-            await message.clear_reactions()
+            try:
+                await message.clear_reactions()
+            except discord.HTTPException as e:
+                if isinstance(message.channel, discord.Thread):
+                    # Threads might not be accessible by the time we try to remove the reaction.
+                    pass
+                else:
+                    raise e
```

Comment:
```suggestion
            except discord.HTTPException:
                # Threads might not be accessible by the time the reaction is removed, so the error is suppressed.
                if not isinstance(message.channel, discord.Thread):
                    raise
```
Not a fan of using pass as you can see ^^

Reply:
I had the condition that way to clarify that the comment belongs to the condition when it's true. I guess it's fine though

Reply:
28574243